lib/src/mt_ptp.c

```diff
@@ -908,11 +1056,16 @@ static int ptp_stat(void* priv) {
     return 0;
   }
 
-  if (ptp->stat_delta_cnt)
+  if (ptp->stat_delta_cnt) {
+    if (ptp->phc2sys.stat_delta_max < 300 && ptp->phc2sys.stat_delta_max > 0) {
+      ptp->phc2sys.stat_sync = true;
```

Comment:
Can you move `ptp->phc2sys.stat_sync = true;` to phc2sys_adjust function. `ptp_stat` is just for debug usage and may be disabled in an production environment. And seems it's duplicate to `enum servo_state`? `JUMP` and `LOCKED` are both a sync stat, right?

Reply:
> Can you move `ptp->phc2sys.stat_sync = true;` to phc2sys_adjust function. `ptp_stat` is just for debug usage and may be disabled in an production environment. And seems it's duplicate to `enum servo_state`? `JUMP` and `LOCKED` are both a sync stat, right?

The `ptp->phc2sys.stat_sync` will be the required condition of starting TSN pacing transmission, the feature will be uploaded later. The `enum servo_state` will not be exposed to video transmitter. So, `ptp->phc2sys.stat_sync` is required.

Reply:
Ok, thanks. Can you move `ptp->phc2sys.stat_sync = true;` to phc2sys_adjust or other palce? This function is only for debug usage.

Reply:
> Ok, thanks. Can you move `ptp->phc2sys.stat_sync = true;` to phc2sys_adjust or other palce? This function is only for debug usage.

Done.